classes/autotools.oeclass

```diff
@@ -48,7 +48,13 @@ autotools_configure[emit] = "do_configure"
 do_configure () {
     do_configure_autotools
 }
+AUTOCONFDIR = ""
+UPDATE_GNUCONF = ""
+UPDATE_GNUCONF:TARGET_CPU_aarch64 = "rm -f ${S}${AUTOCONFDIR}/config.{guess,sub}; cp ${TOPDIR}/meta/core/scripts/config.{guess,sub} ${S}${AUTOCONFDIR}"
```

Comment:
What do you mean by "move this "one-liner" into do_configure_autotools" it's used inside the do_configure_autotools?